ratpack-core/src/test/groovy/ratpack/path/PathBindingSpecSpec.groovy

```diff
@@ -53,4 +54,31 @@ class PathBindingSpecSpec extends RatpackGroovyDslSpec {
     getText("a/b/a/b/xxx") == ":foo/:bar/:foo/:bar/xxx"
   }
 
+  def "binding incorrectly formatted paths to tokens should not cause server errors"() {
+    when:
+    handlers {
+      get(":token") {
+        response.send()
+      }
+    }
+
+    then:
+    statusCode(applicationUnderTest, "networking%party") != 500
```

Comment:
No, you can't. If you could we would do that. :)

The problem is that `TestHttpClient` will not let you send request for illegal paths by throwing an exception, but there is nothing in the http protocol to prevent you from sending illegal paths which is what we are doing in `statusCode()`.

Maybe this justifies getting `TestHttpClient` to allow sending bad paths like these using some new method?